### DISCUSSION TITLE
Renovatebot not picking up new versions of adoptopenjdk image

### DISCUSSION THREAD

**Discussion by vbode** [renovatebot/renovate#9335] [Request Help] [answered]
Hi,

We have a dockerfile setup with this image:
adoptopenjdk:11.0.9_11-jre-openj9-0.23.0

There is now this newer image available:
adoptopenjdk:11.0.10_9-jdk-openj9-0.24.0

When running renovatebot on a repo with this docker image, renovate does not see the newer image. Probably because of the weird version scheme of the image. 

Can something be done about this? Did not see a relevant configuration option, but I might be missing something.

**Comment by viceice** [comment, marked as answer]
Try following package rule
```json
    {
      "description": "Use loose versioning for adoptopenjdk images",
      "matchPackageNames": ["adoptopenjdk"],
      "versioning": "loose",
      "allowedVersions": "/^\\d+\\.\\d+\\.\\d+_\\d+-jdk-openj9-\\d+\\.\\d+\\.\\d+$/"
    }
```

**Reply by vbode** [reply]
Awesome, that did the trick!
Many thanks :)

PS: for posterity, above example uses jdk rather than jre
